plugins/ffmpeg_plugin/kahawai.c

```diff
@@ -203,8 +204,14 @@ static int kahawai_read_header(AVFormatContext* ctx) {
     av_log(ctx, AV_LOG_ERROR, "Only yuv422p10le is supported\n");
     return AVERROR(EINVAL);
   }
+
   ops_rx.transport_fmt = ST20_FMT_YUV_422_10BIT;
-  ops_rx.output_fmt = ST_FRAME_FMT_YUV422RFC4175PG2BE10;
+
+  if (s->ext_frames_mode) {
+    ops_rx.output_fmt = ST_FRAME_FMT_Y210;
```

Comment:
Why not ST_FRAME_FMT_YUV422PLANAR10LE? I see the only support av fmt in the code now is AV_PIX_FMT_YUV422P10LE. Further more it should setup a mapping tables between the av fmt and enum st_frame_fmt.